.github/workflows/cover.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-go@v3
       with:
-        go-version: '^1.19'
+        go-version: '1.19'
```

Comment:
Note to reviewers: Current codecov failures are caused by the most recent version of Go. This change fixes that.

https://github.com/kubernetes-sigs/cluster-api-provider-azure/actions/workflows/cover.yaml